staff/francesc-destruels/precourse/bingo/index.js

```diff
@@ -0,0 +1,148 @@
+// Crea un array con todos los numeros de 1 al 90 ordenados aleatoriamente. Cada turno se mira el siguiente.
+function generateNumberByTurn(){
+    allNumbers = [];
+    while (allNumbers.length < 90){
+        if (allNumbers.includes(number = Math.floor(Math.random()*(91-1)+1)) === false){
+                allNumbers.push(number);
+        }
+    }
+} 
+
```

Comment:
You coud use the shuffle [Fisher Yates method](https://bost.ocks.org/mike/shuffle/), you would spend less resouces to get an array random numbers.

Reply:
I would look it, but it will be impossible to apply it in less than a couple weeks.